fala/templates/adviser/other_region.html

```diff
@@ -0,0 +1,12 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div class="govuk-grid-column-full">
+    <div id="google_translate_element" class="govuk-!-margin-bottom-6"></div>
+    <h1>The postcode {{ postcode }} is in {{ region }}</h1>
+    <p class="govuk-body">
+      Find out about <a class="govuk-link" href="{{ link }}">Legal Aid in {{ region }}</a>
+      or try a different search
+    </p>
```

Comment:
Where is the change search button? This button when clicked will take the user back to the search page.

Reply:
Its on a different ticket

Reply:
No, that's not what I'm talking about. There is a GDS green button that allows the user to click and change the search. Look at the design on the Jira ticket. 

The ticket right at the top under the "When" says: `When a user selects change search, they are redirected back to the search page (Landing page).`